### PR TITLE
First pass at reviewing principles

### PR DESCRIPTION
**Pull request by partner0**

### CODE REVIEW

File: _tabs/principles.md [hunk 1]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
 order: 0
 ---
 
-**We are a fun bunch of devs passionate about collaborating and building easy to use, valuable products. To do so, we abide by the following princilpes**
-
+**We are a bunch of passionate devs having fun building easy to use and valuable products together. To do so, we follow these princilpes**
```
Comment (RCM7) [inline _tabs/principles.md:7]: typo: principles

File: _tabs/principles.md [hunk 2]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
+_"Ship Sooner, Learn Faster: Early delivery for quicker feedback."_
+
+- What to do
+   - Releasing minimum viable products (MVPs) to get feedback from users quickly.
+   - Setting aggressive but realistic release targets to avoid endless development cycles.
+   - Prioritizing time-to-market over exhaustive development, even if it means the first version has known limitations.
+
+- What to avoid
+   - Waiting for a product to be "perfect" before releasing it.
+   - Ignoring early customer feedback or postponing releases due to minor issues.
```
Comment (RCM7) [inline _tabs/principles.md:32]: I don't follow how "ignoring early customer feedback" relates to "postponing releases due to minor issues". Also the latter kinda hints at the same thing as the "waiting for a product to be "perfect" before releasing it

File: _tabs/principles.md [hunk 3]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
+   - Spending too much time on internal polishing without shipping anything.
```
Comment (RCM7) [inline _tabs/principles.md:33]: also seems redundant. Unless that's the goal (say same thing in 3 different ways)

File: _tabs/principles.md [hunk 4]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
 
-## We iterate and improve
-   - Once you got the feedback, go back and improve. As we ship and iterate, we plan for the future as well
-   - We measure what we ship to iterate and improve based on data
+## We learn, iterate, and improve
+_"Learn Fast, Adapt Faster: Every step teaches us something new.""_
```
Comment (RCM7) [inline _tabs/principles.md:36]: I think in this principle we want the opposite of fast. We want to encourage people to actually take the time necessary to improve things. 

Also, I'd keep the "We iterate and improve", and keep the learning part only in the sub-text

File: _tabs/principles.md [hunk 5]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
+
+## We drive revenue
+_"Build for Impact: Create what counts and drives revenue."_
 
-## We deliver value
-   - At the end of the day we are here to make Beyond a successful business and drive revenue
-   - As we focus on our customers and their experience we are bound to do so
+- What to do
+   - Prioritizing features and fixes that have a direct impact on revenue growth, such as improving customer retention or enabling new sales opportunities.
+   -  Collaborating with sales, marketing, and product teams to understand how engineering can best support revenue goals.
```
Comment (RCM7) [inline _tabs/principles.md:66]: typo: extra space in the beginning

File: _tabs/principles.md [hunk 6]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
+   - Measuring the impact of releases on key revenue metrics and making adjustments to improve outcomes.
 
+- What to avoid
+   - Focusing too much on technical elegance or side projects that don’t have a clear connection to revenue.
```
Comment (RCM7) [inline _tabs/principles.md:70]: typo: or -> on

File: _tabs/principles.md [hunk 7]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
 
 ## We take ownership
-   - Owning the code we shipped as we are the expert and knows best
-   - Sharing your knowledge by documenting, pairing and training, because sharing is caring
+_"Own It, Lead It: Great outcomes start with taking responsibility."_
+
+- What to do
+   - Owning the entire lifecycle of our code, from development to production and maintenance.
+   - Actively engaging in problem-solving, taking responsibility for identifying and fixing issues rather than passing them off.
+   - Communicating proactively about project status, blockers, or failures, and following through on commitments.
```
Comment (justipaula) [inline _tabs/principles.md:54]: about project status, blockers, "tradeoffs" or failures

File: _tabs/principles.md [hunk 8]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
+
+- What to do
+   - Owning the entire lifecycle of our code, from development to production and maintenance.
+   - Actively engaging in problem-solving, taking responsibility for identifying and fixing issues rather than passing them off.
+   - Communicating proactively about project status, blockers, or failures, and following through on commitments. 
+
+- What to avoid
+   - Blaming other teams or external factors for project delays or failures.
+   - Failing to address production issues promptly or waiting for someone else to solve the problem.
+   - Over-relying on PMs or others for validation without taking responsibility for writing CI tests.
```
Comment (justipaula) [inline _tabs/principles.md:59]: Over-relying on PMs or others for validation without taking responsibility for writing CI tests or reviewing own deployed chunks.

File: _tabs/principles.md [hunk 9]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
+**We are a bunch of passionate devs having fun building easy to use and valuable products together. To do so, we follow these princilpes**
 
 ## We reduce scope
-   - The best way to ship more stuff is to ship smaller!
-   - The best way to ship safely is to ship often!
+_"Focus Narrow, Deliver Big: Less really is more."_
+
+- What to do
+   - Breaking down large features into smaller, manageable chunks to ensure quicker delivery.
+   - Prioritizing must-have features over nice-to-have ones to focus on core functionality.
```
[the remark this answers is not in the document]
Comment (justipaula) [reply _tabs/principles.md:14]: just would change the end to "to focus on the things that add the most value".

File: _tabs/principles.md [hunk 10]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
-
+**We are a bunch of passionate devs having fun building easy to use and valuable products together. To do so, we follow these princilpes**
 
 ## We reduce scope
-   - The best way to ship more stuff is to ship smaller!
-   - The best way to ship safely is to ship often!
+_"Focus Narrow, Deliver Big: Less really is more."_
+
+- What to do
+   - Breaking down large features into smaller, manageable chunks to ensure quicker delivery.
```
[the remark this answers is not in the document]
Comment (RCM7) [reply _tabs/principles.md:13]: not sure what you mean @justipaula :P

File: _tabs/principles.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (RCM7) [reply _tabs/principles.md:14]: I like core functionality. Feel like "most value" is better used down in the drive revenue

File: _tabs/principles.md [hunk 11]
```diff
@@ -4,26 +4,69 @@ icon: fas fa-heartbeat
...
 
 ## We ship early
-   - Getting feedback as early as possible even if the project/feature isn’t 100% there
-   - While we control radius with feature flags to reduce risk and deliver value to our customers
+_"Ship Sooner, Learn Faster: Early delivery for quicker feedback."_
+
+- What to do
+   - Releasing minimum viable products (MVPs) to get feedback from users quickly.
+   - Setting aggressive but realistic release targets to avoid endless development cycles.
+   - Prioritizing time-to-market over exhaustive development, even if it means the first version has known limitations.
```
[the remark this answers is not in the document]
Comment (partner0) [reply _tabs/principles.md:28]: Agreed, except for re-usability. Always thinking about reusability is a trap, when a majority of the code you write will never be re-used. That can lead to abstraction hell. Only abstract when you need to, or when you know you will re-use or need to abstract, like the integrations framework for eg. I personally fell into that trap way too often 😅

**Pull request merged**